cmd/cli/commands/infrastructure.go

```diff
@@ -147,3 +149,70 @@ func validateFilePath(path string) error {
 	}
 	return nil
 }
+
+// DeleteInstancesCmd returns a command that deletes instances
+func DeleteInstancesCmd() *cobra.Command {
```

Comment:
Why did we need to create a new delete cmd? There is already `deleteInfraCmd` which is deleting the instance. 

We could rename the infra commands to instance commands for clarity.